GOVERNANCE.md

```diff
@@ -0,0 +1,114 @@
+
+## Organizational Structure
+
+This section defines the teams and roles of the Pulsar team itself.
+
+### GitHub
+
+Our primary (and currently only) official repository organization is the [Pulsar-Edit Organization](https://github.com/pulsar-edit) on GitHub.
+To run this effectively we have a number of sub-teams to perform various duties across our various repositories and other areas contained on the site.
+
+#### Admin team (Owners)
+
+Responsible for GitHub organization-wide duties such as:
+- Team and role administration
+- Repository configuration & administration
+
+#### Sub-Teams (Members)
+
+Responsible for the day to day management of repositories they are assigned to.
+- Reviewing and merging PRs
+- Triaging issues
+- Maintaining project boards
+
+Team members may be assigned to more than one team.
+
+The current teams on GitHub are as follows:
+- **Core** - works on the Main part of our editor
+- **Backend** - API and server Development
+- **Documentation** - Website and Documentation team
+- **Packages** - Works on our core packages
+
+### Discord
+
+Our Discord server is the preferred communication method for team conversations and decision-making.
+We define a number of official roles on here (as well as community ones) to make sure that administration and moderation requirements are met.
+
+#### Admins
+
+Responsible for overall configuration and maintenance of the server.
+- Creating, editing, removing channels
+- Administrating roles and members
+- Maintaining bot and app integrations
+- Kicking/Banning users that break our rules and guidelines and code of conduct.
+
+#### Moderators
+
+Responsible for the day to day moderation of the server and community interactions.
+- Ensuring civil discourse is taking place and taking action (warning users etc.)
+- De-escalating any situations or heated conversations
+
+#### Community roles
+
+Community roles can be self-granted via the `#role-select` channel.
+These members are people who are interested or participate within the Pulsar community and are expected to adhere to our code of conduct but otherwise do not have any official or specific responsibilities.
+These roles are designed to help people get updates and communicate within the specific areas of the project that they are interested in.
+Pulsar team members will also be part of these groups but they may not entirely correlate to the memberships and responsibilities granted in the GitHub sub-teams.
+These roles are:
+
+##### Core teams
+- Core - works on the Main part of our editor
+- Backend - API and server Development
+- Documentation - Website and Documentation team
+- Packages - Works on our core packages
+- Developer - Just fills in anywhere that they want
+
+##### Operating Systems
+- Linux
+- MacOS
+- Windows
+
+##### Other roles
+- Community Packages - join if you're into making packages for us
```

Comment:
```suggestion
- Community Packages - join if you're into making packages for the editor
```

Reply:
That bit was taken verbatim from discord. I mean, I agree with the change but I think it should be changed there too.

Reply:
Great catch there, this does sound better